common/config/jest/jestSetup.js

```diff
@@ -10,3 +10,6 @@ console.error = (...args) => {
 console.warning = (...args) => {
   throw args;
 };
+
+// Need ResizeObserver for GridLayout
+global.ResizeObserver = require('resize-observer-polyfill');
```

Comment:
Why do we need this?

Reply:
For some reason, storybook does not have access to the typescript library's `ResizeObserver` and I get this failure when I run the snapshot tests:
![image](https://user-images.githubusercontent.com/79475487/135697448-eb2e9105-3f23-4898-b68f-a0c57825ef64.png)

Reply:
Can you update the comment with this? Something like:
```
Add `ResizeObserver` to globals. Without this jest tests fail with "ReferenceError: ResizeObserver is not defined"
```